modules/update-duration-example.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/understanding-openshift-update-duration.adoc
+
+:_content-type: CONCEPT
+[id="update-duration-example_{context}"]
+= Example update duration of cluster Operators
+
+image::update-duration.png[A diagram displaying the periods during which cluster Operators update themselves during an OCP update]
+
+The previous diagram shows an example of the time that cluster Operators might take to update to their new versions.
+The example is based on a three-node AWS OVN cluster, which has a healthy compute `MachineConfigPool` and no workloads that take long to drain, updating from 4.13.0 to 4.14.0.
+
+[NOTE]
+====
+The specific update duration of a cluster and its Operators can vary based on several cluster characteristics, such as the target version, the amount of nodes, and the types of workloads scheduled to the nodes.
+====
+
+Each cluster Operator has characteristics that affect the time it takes to update itself.
+For instance, the Kube API Server Operator in this example took over eleven minutes to update because `kube-apiserver` provides graceful termination support, meaning that existing, in-flight requests are allowed to complete gracefully.
```

Comment:
```suggestion
For instance, the Kube API Server Operator in this example took more than eleven minutes to update because `kube-apiserver` provides graceful termination support, meaning that existing, in-flight requests are allowed to complete gracefully.
```

Reply:
>  over eleven minutes

Is this a particularly long period of time to update?